ubuntu.sh

```diff
@@ -36,16 +36,12 @@ init_system() {
 
 internal_ip() {
   ip addr \
-    | grep "$(internal_iface)" \
-      | grep "inet 10\." \
+    | grep "$INTERNAL_IFACE" \
```

Comment:
Does `INTERNAL_IFACE` get set on other providers? If so, consider changing line 19 to account for that (keep DO from breaking)